config/install/core.entity_form_display.node.alert.default.yml

```diff
@@ -25,56 +27,60 @@ third_party_settings:
       children:
         - field_alert_visibility_pages
         - field_alert_visibility_state
+      label: 'Visibility pages'
+      region: hidden
       parent_name: ''
-      weight: 19
+      weight: 23
       format_type: fieldset
       format_settings:
-        id: ''
         classes: ''
+        id: ''
         description: 'Control where alert should be displayed or not.'
         required_fields: true
-      label: 'Visibility pages'
-      region: hidden
+_core:
+  default_config_hash: E1tr4PZ8_KHdG29r71AECUxwNLjOFX2BfHG3c0k6LW0
```

Comment:
same for hashes in configs - no go, please

Reply:
@podarok sorry, my bad. Fixed in https://github.com/open-y-subprojects/openy_node_alert/pull/146/commits/d4766904caa668a092eab382a4fc9a3b72c868cc